src/registers/model_specific.rs

```diff
@@ -112,6 +123,29 @@ bitflags! {
     }
 }
 
+bitflags! {
+    /// Flags stored in IA32_U_CET (Table-2-2 in Intel SDM Volume 4). The Intel
+    /// SDM-equivalent names are described in parentheses.
```

Comment:
This comment should also mention `IA32_S_CET`. Other than that I really like the comments for this structure, well done :)